dashboards.tf

```
@@ -398,3 +398,10 @@ module "orange_sla_dashboard" {
   source = "./modules/dashboard"
   path   = "service-level-agreements/orange.json"
 }
+
+### Capacity & Performance Dashboards ### 
+
+module "performance_application" {
```

Just ensure that the formatting is correct here, like the spacing...
```suggestion
module "performance_application" {
  source = "./modules/dashboard"
  path   = "capacity/application-performance-review.json"
}
```